plugins/source/heroku/client/spec.go

```diff
@@ -31,36 +20,18 @@ type BackoffSettings struct {
 
 func (c Spec) ClientOptions() []option.ClientOption {
```

Comment:
```suggestion
func (Spec) ClientOptions() []option.ClientOption {
```

Reply:
removed the 'c' completely since '_' also complained about being unused :)